apps/docs/pages/guides/ai/examples/image-search-openai-clip.mdx

```diff
@@ -97,7 +97,7 @@ def seed():
 
     # add records to the *images* collection
     images.upsert(
-        vectors=[
+        records=[
```

Comment:
Not needed!

Reply:
but .upsert in python vecs package doesn't accept vectors= as arguments only records?